log4j-core/src/test/java/org/apache/logging/log4j/core/layout/JsonLayoutTest.java

```diff
@@ -604,6 +608,37 @@ public void testEmptyValuesAreIgnored() {
         assertFalse(str, str.contains("\"empty\""));
     }
 
+    @Test // LOG4J2-3358
+    public void jsonLayout_should_substitute_lookups() {
```

Comment:
```suggestion
    /**
     * @see <a href="https://issues.apache.org/jira/browse/LOG4J2-3358">LOG4J2-3358</a>
     */
    @Test
    public void ctx_lookup_should_work() {
```

Reply:
Structure borrowed from the test above: https://github.com/apache/logging-log4j2/blob/e7af4e0a1ead739598a5ca96d09bc214b68cf0ec/log4j-core/src/test/java/org/apache/logging/log4j/core/layout/JsonLayoutTest.java#L598-L599

Happy to change both